app/Models/Application.php

```
@@ -864,6 +865,13 @@ public function nixpacks_environment_variables()
             ->where('key', 'like', 'NIXPACKS_%');
     }
 
+    public function railpack_environment_variables()
+    {
+        return $this->morphMany(EnvironmentVariable::class, 'resourceable')
+            ->where('is_preview', false)
+            ->where('key', 'like', 'RAILPACK_%');
+    }
```

PHPMD (2.15.0)

[error] The method railpack_environment_variables is not named in camelCase. (undefined)

(CamelCaseMethodName)